data/json/recipes/food/frozen.json

```diff
@@ -170,7 +176,7 @@
     "qualities": [ { "id": "COOK", "level": 3 } ],
     "tools": [ [ [ "rock_quern", -1 ], [ "clay_quern", -1 ], [ "mortar_pestle", -1 ], [ "food_processor", 10 ] ] ],
     "components": [
-      [ [ "milk_standard", 2, "LIST" ] ],
+      [ [ "milk_standard", 2, "LIST" ], [ "lf_milk_standard", 2, "LIST" ] ],
```

Comment:
This is possible but requires another source of fat.
https://www.masterclass.com/articles/almond-milk-ice-cream-recipe#3-ways-to-customize-almond-milk-ice-cream